keyboards/handwired/dactyl_manuform/5x6_68/keymaps/default/keymap.c

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Kostas Pagratis (@kpagratis)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum custom_layers {
+    _QWERTY,
+    _LAYER2,
+};
+
+#define GUI_J   RGUI_T(KC_J)
+#define GUI_F   LGUI_T(KC_F)
+#define SFT_A   LSFT_T(KC_A)
+#define CTL_S   LCTL_T(KC_S)
+#define ALT_D   LALT_T(KC_D)
+#define ALT_K   RALT_T(KC_K)
+#define CTL_L   RCTL_T(KC_L)
+#define SFT_SC  RSFT_T(KC_SCLN)
+#define IMPORTS LALT(LCTL(KC_O))    //Intellij Organize Imports
+#define LINES   LALT(LGUI(KC_L))    //Intellij Reformat Code
+#define USAGES  LALT(KC_F7)         //Intelllj Find Usages
+#define RENAME  LSFT(KC_F6)         //Intellij Refactor Rename
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT(
```

Comment:
This should be side by side, not stacked. (and the info.json probably needs to be updated to reflect that, as well.)

Reply:
I'm trying to wrap my head around how to make the halves side by side in the info.json and I can't wrap my brain around how that definition translates to the pinouts on the MCs